src/scales/timeScale.ts

```diff
@@ -47,6 +46,38 @@ export module Scales {
       var startTimeValue = endTimeValue - Plottable.MILLISECONDS_IN_ONE_DAY;
       return [new Date(startTimeValue), new Date(endTimeValue)];
     }
+
+    public scale(value: Date): number {
+      return this._d3Scale(value);
+    }
+
+    protected _getDomain() {
+      return this._d3Scale.domain();
+    }
+
+    protected _setBackingDomain(values: Date[]) {
```

Comment:
Since there's gotta be something backing the domain of the scale but it is never explicitly set anywhere, the domain had to actually be set somehwere.  There is an implementation of `_setDomain()` that has to tap into how to actually set the domain of the scale, and I'm unsure of how this name should be.